server.js

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env node
```

Comment:
Could you explain why? It just only take a path to the `package.json` file and return the binary name, no more. This allow to ask for the correct binary to be executed from `bash` scripts, so I can create [NodeOS](http://node-os.com/) single process images. If you still don't want it I could remove the CLI, but it would make me thing far easier a lot to have it integrated in the module itself...